### PR TITLE
Add the community section in the README.md

### PR DESCRIPTION
**Pull request by Brouilles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +198,13 @@ The Core Admin Razor Class library will then serve the static assets from this U
 
 If you see a 401 error when accessing /coreadmin it means you are running in a non-development environment and you have not set a role up. See "Role based security" above.
 
+### Community
+
+You can find in this section some links about community content (writting, code samples, etc...). Don't hesitate to make a PR to add your own.
```
Comment (edandersen) [inline README.md:203]: writing

**Comment by edandersen** [comment]
One typo then insta-merge!

File: README.md (hunk 1 above)
Comment (Brouilles) [reply README.md]: Sorry!

**Comment by edandersen** [comment]
Thanks!!

**Pull request merged**